telemetry/noop_test.go

```
@@ -0,0 +1,20 @@
+
+package telemetry_test
+
+import (
+	"testing"
+
+	"github.com/corazawaf/coraza/v3/experimental/plugins/plugintypes"
+	"github.com/corazawaf/coraza/v3/telemetry"
+)
+
+func TestNoopSatisfiesInterface(t *testing.T) {
+	var s plugintypes.TelemetrySink = telemetry.Noop()
```

golangci-lint (2.11.4)

[error] ST1023: should omit type plugintypes.TelemetrySink from declaration; it will be inferred from the right-hand side

(staticcheck)